_posts/2016/02/2016-02-03-swdcretreat2015-university_course.html

```diff
@@ -0,0 +1,437 @@
+<a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002204.html'>Ethan White</a>
+and <a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002211.html'>Michael Jackson</a> went on to describe their own experiences with setting up a course.
+I want to believe the push for code sharing and reproducible articles from
+<a href='http://www.nature.com/news/interactive-notebooks-sharing-the-code-1.16261'>Nature</a> and the submission guidelines for
+<a href='http://journals.plos.org/plosone/s/materials-and-software-sharing'>PLOS ONE</a>
+have addressed <a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002205.html'>Greg's</a> initial observations that universities find that lab skills course is not worth graduate credit.  The conversation continues
+<a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002207.html'>here</a>,
+<a href='http://lists.software-carpentry.org/pipermail/discuss_lists.software-carpentry.org/2014-September/002210.html'>here</a>, and
+a blog post by Damien Irving <a href='http://software-carpentry.org/blog/2014/02/university-course.html'>here</a>.
+
+The Software Carpentry + Data Carpentry Instructor & Helper Retreat seemed like a good means to revisit the discussion.
+
+<h2>Who are we?</h2>
+<a href='http://software-carpentry.org/pages/team.html#chen.daniel'>Daniel Chen</a>
+<a href='http://software-carpentry.org/pages/team.html#timbers.tiffany'>Tiffany Timbers</a>, and
+<a href='http://software-carpentry.org/pages/team.html#bryan.j'>Jenny Bryan</a>
+
+<h2>The Roundtable Discussion</h2>
+
+<p>
+  <h4>
+    Earlier this year, Anthony Scopatz and Kathryn Huff published <a href='http://physics.codes/'>Effective Computation in Physics</a> and created a course around the book.
+    Tiffany, can you talk about the <a href='https://phy3009.github.io/PHY3009-2015/'>course</a> you've been beta testing?
+  </h4>
+</p>
+
+<p>
+  [T]: Anthony has been teaching the course to undergraduate students and I've been building off Katy and Anthony's experiences from teaching as well as my own from teaching Software Carpentry.
+  I decided to use this textbook as my course at Quest.
+  The book is based on Python, but many of the principles are language agnostic.
+  The book covers many topics SWC teaches, such as Bash, Python, Make, documentation, licenses, and many open science aspects.
+  Katy and Anthony also created a <a href='https://github.com/physics-codes/examples'>Github repo</a> of annotated iPython (Jupyter) notebooks of the code in the book, as well as an <a href='https://github.com/physics-codes/seminar'>example syllabus</a>.
```

Comment:
please change the url here to: https://github.com/physics-codes/seminar, and remove ", as well as an <a href='https://github.com/physics-codes/seminar'>example syllabus</a>" as there is no syllabus.